.github/env-workflow.yml

```diff
@@ -9,4 +9,6 @@ dependencies:
   - lxml
   - pyRestTable
   - Sphinx
+  - nbsphinx
```

Comment:
Will there be a step added to the workflow (and also to the `/Makefile`) that uses `nbsphinx` to build documentation pages from the notebooks?  There is an [example `Makefile`](https://github.com/bluesky/hklpy/blob/main/examples/Makefile) in the *hklpy* project.

Reply:
That step would happen *before* sphinx is called to `make html`, and then the PDF version of the documentation.

Reply:
There is no need for adding another steps or modifying the makefile. The sphinxs extension `nbsphinx` takes care of everything (basically the .ipynb become equivalent to .rst files).